cfgov/unprocessed/css/molecules/info-unit.less

```diff
@@ -29,7 +29,7 @@
   &_heading-link {
```

Comment:
L10 in this file needs `&_image` changed to `&__image`.

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/8274